app/src/main/java/app/insti/fragment/ComplaintDetailsFragment.java

```diff
@@ -99,7 +99,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
 
         initialiseViews(view);
         upVotesList = new ArrayList<>();
-        commentListAdapter = new CommentsAdapter(getContext(), sId, uId, this);
+        commentListAdapter = new CommentsAdapter(getActivity(),getContext(), sId, uId, this);
```

Comment:
Space after comma. This looks ugly. Also, I hardly think you need _both_ references (activity and context) anywhere.